lms/static/sass/shared/_footer.scss

```diff
@@ -5,6 +5,7 @@
 
 .wrapper-footer {
   @extend %ui-print-excluded;
+  margin-top: ($baseline*2);
```

Comment:
I put this in the general footer because it is broader than the layouts file - this is a fix for the all page footers, not just the ones with responsive layouts.